conformance/Makefile

```diff
@@ -89,7 +89,7 @@ uninstall-nkg: uninstall-k8s-components undo-manifests-update ## Uninstall NKG o
 
 .PHONY: uninstall-k8s-components
 uninstall-k8s-components: ## Uninstall installed components on configured kind cluster
-	kubectl delete -f $(NKG_MANIFEST)
+	-kubectl delete -f $(NKG_MANIFEST)
```

Comment:
?

Reply:
I noticed when I was running the conformance tests that we were getting an error in the teardown because the CRD is deleted before the resource delete is called, which meant the rest of the make command wasn't completing. The `-` means the error won't stop the next commands in the target from running.

Reply:
hm. but CRDs are removed in the next line, no?

Reply:
No, I believe the next line reverts any gateway API version changes (e.g. if we are running against the sig main branch etc.) It's the deletion of the new NKG CRD that is causing the error.